src/api/hooks/index.ts

```diff
@@ -0,0 +1,2 @@
+import useCoverVideo from './coverVideo'
```

Comment:
Let's be consistent and move this to `src/hooks`

Reply:
I believe this is a right place for that. `src/hooks` will be used for utility/logic related hooks. This is strictly related to the data fetching. Right now, because of query node restrictions, we return mocked value from `useCoverVideo` anyway, but once that's alleviated, we will do Apollo query inside it, so it makes sense to live in `api`